pages/releases/4.3.html

```diff
@@ -0,0 +1,1944 @@
+
+				<div id="rendering-compositor-effects-api" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#rendering-compositor-effects-api">Compositor effects API</a>
+								</h4>
+								<div class="c-content">
+									<p class="replace-me"></p>
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #rendering-compositor-effects-api -->
+
+				<div id="rendering-renderingdevice-context-management-refactor" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#rendering-renderingdevice-context-management-refactor">
+										RenderingDevice context management refactor
+									</a>
+								</h4>
+								<div class="c-content">
+									<p class="replace-me"></p>
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #rendering-renderingdevice-context-management-refactor -->
+
+				<div id="rendering-depth-based-fog" class="release-card media-left span-2">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#rendering-depth-based-fog">Depth-based fog</a>
+								</h4>
+								<div class="c-blockquote">
+									<span class="highlight">Hide</span> and see?…
+								</div>
+								<div class="c-content">
+									<p>
+										With depth-based fog, you can now decide when the fog begins and when it is
+										fully opaque.
+									</p>
+									<p>
+										Compared to exponential (and realistic) fog mode, it may seem less realistic,
+										but you now have the tools to create an atmosphere worthy of old-school horror
+										games.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="scriptsengineer"
+									data-read-more="https://github.com/godotengine/godot/pull/84792">
+								</div>
+							</div>
+						</div>
+						<div class="release-card-media">
+							<video class="release-card-video" autoplay loop muted
+								poster="/storage/releases/4.3/images/depth-fog.webp">
+								<source src="/storage/releases/4.3/video/depth-fog.webm" type="video/webm">
+							</video>
+						</div>
+					</div>
+				</div> <!-- #rendering-depth-based-fog -->
+
+				<div id="rendering-compatibility-rendering-backend-features" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#rendering-compatibility-rendering-backend-features">
+										Compatibility rendering backend features
+									</a>
+								</h4>
+								<div class="c-content">
```

Comment:
The Compatibility rendering backend received a lot of attention this release cycle. We now consider it feature complete as it has all the features we intended it to have. 

New in this release are MSAA, resolution scaling, Glow, ReflectionProbes, LightmapGI, Adjustments, and Color Correction.

Reply:
PRs:
MSAA + scaling: https://github.com/godotengine/godot/pull/83976
Glow: https://github.com/godotengine/godot/pull/87360
ReflectionProbes: https://github.com/godotengine/godot/pull/88056
LightmapGI: https://github.com/godotengine/godot/pull/85120
Adjustments and Color correction: https://github.com/godotengine/godot/pull/91176

Contributors: Bastiaan Olij, David Snopek, Clay John